src/Files.App/Actions/Open/EditInNotepadAction.cs

```diff
@@ -20,7 +20,9 @@ public RichGlyph Glyph
 
 		public bool IsExecutable =>
 			context.SelectedItem is not null &&
```

Comment:
```suggestion
			context.SelectedItems.Any() &&
```
This should fix the issue with multiple selection.